Added Mentor session Test case and logging call Log on screenshot

ShreeJithR opened the pull request:
- Mentor Session Creation(BBB),Mentee enrolling and joining the session and providing feedback.
- Mentor Session Creation(BBB-->Gmeet) Mentee enroll and unenroll to verify Mentee count and list.

ShreeJithR committed:
Added Mentor session Test case

-Mentor Session Creation(BBB),Mentee enrolling and joining the session and providing feedback
- Mentor Session Creation(BBB-->Gmeet) Mentee enroll and unenroll to verify Mentee count and list

infa-rvarahag commented on src/test/java/org/shikshalokam/Listerners/SLCustomListener.java:
1) Logger.info  not      logger.error

infa-rvarahag commented on src/test/java/org/shikshalokam/frontend/TestMentorSession.java:
Class Name should be TestMentorSessionCreation  same way Method names testmentorSessionBBBCreation, testmentorSessionGmeetMenteeCount

infa-rvarahag commented on src/test/java/org/shikshalokam/uiPageObjects/AppCreateSessionPage.java:
Is really needed this sleep ?
if needed overall Just move to the  verifyToastMessage method .
So that all the places we can avoid try catch blocks simply .

infa-rvarahag commented on src/test/java/org/shikshalokam/uiPageObjects/AppSessionDetailsPage.java:
Can you please use https://playwright.dev/java/docs/test-assertions

infa-rvarahag commented on src/test/java/org/shikshalokam/uiPageObjects/PWBasePage.java:
method arguments  need to change : Avoid taking Exception as argument.
logic need to be finetuned   and system outs need to be removed

ShreeJithR committed:
Resolved 3 comments

pending with 2 comments
i.e  related to Listerners and capturing screenshot method

ShreeJithR committed:
Resolved comments

- using waitfor(new Locator.WaitForOptions().setTimeout(10000)); wherever applicable

ShreeJithR committed:
removed Test(description)

ShreeJithR committed:
Minor changes

- replaced System outs with logger info
- replaced sleep with page.waitForTimeout()
- removed duplicate methods which were already present in respective PageObjects
